capstone/capdb/models.py

```diff
@@ -1662,10 +1663,15 @@ class ExtractedCitation(models.Model):
     reporter_name_original = models.CharField(max_length=200)
     volume_number_original = models.CharField(max_length=64, blank=True, null=True)
     page_number_original = models.SmallIntegerField(null=True, blank=True)
+    normalized_cite = models.SlugField(max_length=10000, null=True, db_index=True)
```

Comment:
I think we drop the db_index on `cite_original` when adding this, to make inserts faster?